client/my-sites/stats/feedback/modal/index.tsx

```diff
@@ -20,6 +22,20 @@ const FeedbackModal: React.FC< ModalProps > = ( { siteId, onClose } ) => {
 	const dispatch = useDispatch();
 	const [ content, setContent ] = useState( '' );
 
+	const {
+		data: isAbleToSubmitFeedback,
+		isFetching: isCheckingAbilityToSubmitFeedback,
+		refetch: refetchNotices,
+	} = useNoticeVisibilityQuery( siteId, 'able_to_submit_user_feedback' );
+
+	// Disable feedback submission for 24 hours.
+	const { mutateAsync: disableFeedbackSubmissionInOneDay } = useNoticeVisibilityMutation(
+		siteId,
+		'able_to_submit_user_feedback',
+		'postponed',
+		24 * 3600
```

Comment:
The `able_to_submit_user_feedback` string is used twice so it might be better to make it a const value. I'm not entirely sure what the string represents so that could be clarified by the const definition.

```
const FEEDBACK_NOTICE_KEY = 'feedback-action-leave-review';
```

Reply:
Changed to

```
const NOTICE_KEY_TO_DISABLE_FEEDBACK_SUBMISSION = 'able_to_submit_user_feedback';
```